pylint/checkers/variables.py

```diff
@@ -2671,6 +2669,7 @@ def _check_unpacking(
 
         # Attempt to check unpacking is properly balanced
         values = self._nodes_to_unpack(inferred)
+        details = _get_unpacking_extra_info(node, inferred)
```

Comment:
this prob should've been below the if statement in ln 2675 but no big deal, I'll move it in the next PR

Reply:
nvm, I see it's used below!

Reply:
No I thought of it too (and then forgot). Performance wise this is pretty bad, we'd better duplicate the code so we only create the details when something is going to be raised.